Highlights 2.2.0 (#13411)
WalletWasabi/WalletWasabi: Contrib/ReleaseHighlight.md

## Release Highlights

🥕 Taproot receive addresses
🚀 Faster transaction broadcasting
⏫ More accurate CPFP fee estimation
📉 Safer protocol

> 

Reviewer: `Backend optimizations` should be added here?

## Release Highlights

🥕 Taproot receive addresses
🚀 Faster transaction broadcasting
⏫ More accurate CPFP fee estimation
📉 Safer protocol
🪲 Backend optimizations

## Release Summary

Wasabi Wallet v2.2.0.0 release

> This version of Wasabi is full of extra features, improvements, and fixes. Here’s what got updated in today’s release:

Reviewer: ```suggestion
This version of Wasabi is full of extra features, improvements, and fixes. The highlights are:
```

🥕 Taproot receive addresses

Users can now choose taproot when generating a new receiving address. Spending a taproot input is 16% cheaper compared to spending a native segwit input. Users aren’t the only ones that benefit - Bitcoin nodes can verify taproot’s signatures faster than legacy ones, so the entire network gets a little speed boost.

🚀 Faster transaction broadcasting

> Spending your coins is not only cheaper in this release, it’s also faster! Transactions are now broadcast to multiple nodes in parallel (through the Tor network) and fail more quickly if the transaction is invalid. Users who upgrade will notice it now takes half as long to send their coins.

Reviewer: ```suggestion
A more sophisticated transaction broadcasting mechanism was introduced to have faster transaction broadcasting. Transactions are now broadcast to multiple nodes in parallel (through the Tor network) and fail more quickly if the transaction is invalid. Users who upgrade will notice it now takes half as long to send their coins.
```

nice marketing phrase. seems a bit over exaggerated imo. As it's only the case if people use the taproot addr and even if they do they they will probably not notice the few sats savings.

`if the transaction is invalid`
this can be confusing what is meant here, maybe clarify

Reviewer 2: I agree it was too enthusiastic for me as well, I had to think 5s about how it get cheaper then was disappointed :D

⏫ More accurate CPFP fee estimation

> The CPFP feature now takes into account the fee paid by the parent transaction. In previous releases, the child transaction assumed the parent paid 0 sats in fees. This improvement will be especially significant in high-fee environments, where the CPFP feature is most useful!

Reviewer: ```suggestion
The CPFP feature now takes into account the fee paid by the parent transaction. In previous releases, the child transaction assumed the parent paid 0 sats in fees and thus usually overpaid for the speed up. This improvement will be especially significant in high-fee environments, where the CPFP feature is most useful!
```

Reviewer 2: It's not usually, it's always

Reviewer: true, with "usually" I mean that fees might go up suddenly and then you're just lucky the "bug" made you pay just enough to get confirmed

📉 Safer protocol

> In order to avoid introducing unknown incentives and to limit risk to users, the coordination fee concept has been removed. Free to use coinjoin coordinators are supported.

Reviewer: ```suggestion
In order to avoid introducing unknown incentives and to limit risk to users, the coordination fee concept has been removed. Coinjoin coordinators that charge a coordination fee of 0 continue to be supported.
```
maybe this?, prevent misconception that it's free (mining fees)

or _Coinjoin coordinators that don't charge a coordination fee continue to be supported_
but I think mentioning `0` is easier as the explorers also have this.

Reviewer 2: I reworked your sentence, it is better yes
